CHANGES.md

```diff
@@ -13,6 +13,7 @@
   error messages (#176, @gpetiot)
 - Use the 'user' field in the configuration before inferring it from repo URI
   and handles HTTPS URIs (#183, @gpetiot)
+- Pkg.find_file should ignore backup files (#194, @gpetiot)
```

Comment:
When adding entries to the changelog,we should try to make them accessible to our users. They probably have no idea what `Pkg.find_file` is or what it does as it's just some implementation detail.

Something like `Ignore backup files` is probably enough. Might eventually be worth listing the files that are impacted, `CHANGES`, `LICENSE`, etc...

Reply:
done